vsce/Makefile

```diff
@@ -0,0 +1,17 @@
+# We can derive Reach's Visual Studio Code extension's
+# version using the VERSION file, which can be found
+# in the parent folder.
+include ../VERSION
+
+# the next line is so this Makefile can do arithmetic
+SHELL=/bin/bash
+
+# double dollar sign to have the shell do the expansion
+# https://stackoverflow.com/a/15978322
+VSCE_VER := $$(($$(($$(($(MAJOR)+1))*1000)) + $(MINOR))).$(PATCH).$(RC_VERSION)
+
+# @echo $(VSCE_VER) was very, very helpful for testing
```

Comment:
I get anxious when there are comments like this because I feel like they were useful for your process but they will get out of date and may mislead someday

Reply:
I see. When you say "comments like this", can you be more specific? Are all of the comments potentially dangerous, or just the ones on lines 9-10 and 13?

If you add something about this to The Reach Way of Engineering, I'll read it.

Reply:
6a12c40 removes line 13, `# @echo $(VSCE_VER) was very, very helpful for testing`, as a start. I'll remove any other ones that you say can cause problems long-term.

Reply:
For any future people reading this, https://stackoverflow.blog/2021/07/05/best-practices-for-writing-code-comments/ has some solid suggestions.

Reply:
3cc77279f87636615221b180380f5864129e0cbf follows "Rule 6: Provide links to the original source of copied code".

Let me know if there are other comments I need to fix.